src/app/payment/paymentMethod/DigitalRiverPaymentMethod.tsx

```diff
@@ -45,11 +47,11 @@ const DigitalRiverPaymentMethod: FunctionComponent<DigitalRiverPaymentMethodProp
                 setSubmitted(true);
                 submitForm();
             },
-            onError: (error: Error) => {
-                onUnhandledError?.(error);
+            onError: (_error: Error) => {
```

Comment:
If the argument isn't being used, we can just remove it entirely.

Reply:
Hey @leeBigCommerce, nice catch, I already made the change